plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -238,14 +241,20 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
             override fun getPreconditions(returnVariable: VariableEmbedding) =
```

Comment:
perhaps we can use a builder here, similarly at what has been done for predicates creation. WDYT?

Reply:
Sure, for one there is already a list builder in the standard library